dpnp/backend/include/dpnp_gen_2arg_3type_tbl.hpp

```diff
@@ -184,9 +184,9 @@ MACRO_2ARG_3TYPES_OP(dpnp_power_c,
 
 MACRO_2ARG_3TYPES_OP(dpnp_subtract_c,
                      input1_elem - input2_elem,
-                     nullptr,
-                     std::false_type,
+                     sycl::sub_sat(x1, x2),
```

Comment:
Why use subract with saturation? I think we should rather do `[](auto x1, auto x2) { return x1 - x2; }`.

Reply:
right, no reason for saturation here, I'll resolve that